### PR TITLE
docs: revamp README

### PR DESCRIPTION
**Pull request by mozwell**
# What's new?
- Revamp README and add a detailed API Documentation
- Add details for write APIs

### CODE REVIEW

File: docs/api/create-instance.md [hunk 1]
```diff
@@ -0,0 +1,77 @@
...
+```javascript
+const { createInstance } = require('dotbit')
+
+// To create a DotBit instance by using default config
+const dotbit = createInstance();
+console.log(dotbit);
+
+// ...
+// The printed result would be like:
+DotBit {
```
Comment (zgayjjf) [inline docs/api/create-instance.md:25]: I think the developers don't care what the printed fully result of a Class instance would be.
They only care about what they can do next.

There are too many irrelevant properties, and this kind of printed value will blur the critical point.

File: docs/api/dotbit.md [hunk 2]
```diff
@@ -0,0 +1,790 @@
...
+### Example
+```javascript
+// To get the signer of current DotBit instance.
+console.log(dotbit.signer)
+
+// ...
+// The printed result would be like:
+EthersSigner {
+  signer: Wallet {
+    _isSigner: true,
```
Comment (zgayjjf) [inline docs/api/dotbit.md]: Same here. The printed result contains something they will never care about.

File: docs/api/dotbit.md [hunk 3]
```diff
@@ -0,0 +1,790 @@
...
+```
+
+## alias
+The API is the same as [reverse(keyInfo)](#reversekeyinfo).
+
+## accountsOfOwner(keyInfo)
+List all .bit accounts (including SubDID accounts) of a given blockchain address
+### Parameters
+- keyInfo: `KeyInfo`
+  - key: `string`. The address on a certain blockchain
```
Comment (zgayjjf) [inline docs/api/dotbit.md:298]: There can be a link to the `coin_type` part of the FAQ.

**Review by zgayjjf — APPROVED** [review]

File: docs/api/create-instance.md (hunk 1 above)
Comment (mozwell) [reply docs/api/create-instance.md:25]: I have different opinions. One important thing is that we need to build trust with external developers. By showing them the return value in a precise way, they would have a good expectation of what they will get once calling our APIs. Besides, we already have a table of contents, by using of which external developers will know how many APIs we have provided, and navigate through these APIs without much effort, so we don't have to worry blurring critical points by providing precise return value.

File: docs/api/dotbit.md (hunk 2 above)
Comment (mozwell) [reply docs/api/dotbit.md]: Same above.

File: docs/api/dotbit.md (hunk 3 above)
Comment (mozwell) [reply docs/api/dotbit.md:298]: Done.

File: docs/api/dotbit.md (hunk 2 above)
Comment (zgayjjf) [reply docs/api/dotbit.md]: `signer`, outside component.

File: docs/api/dotbit.md [hunk 4]
```diff
@@ -0,0 +1,790 @@
...
+      baseUri: 'https://subaccount-api.did.id/v1',
+      net: [Networking]
+    },
+    registerAPI: RegisterAPI {
+      baseUri: 'https://register-api.did.id/v1',
+      net: [Networking]
+    }
+  },
+  signer: EthersSigner {
+    signer: Wallet {
```
Comment (zgayjjf) [inline docs/api/dotbit.md]: `signer`

File: docs/api/dotbit.md [hunk 5]
```diff
@@ -0,0 +1,790 @@
...
```
Comment (zgayjjf) [inline docs/api/dotbit.md]: signer

File: docs/api/dotbit.md [hunk 6]
```diff
@@ -0,0 +1,790 @@
...
```
Comment (zgayjjf) [inline docs/api/dotbit.md]: signer

**Pull request merged**